.github/workflows/ci.yml

```diff
@@ -18,11 +18,15 @@ jobs:
         version:
           - '1.6'
           - '1'
-          - 'nightly'
         os:
           - ubuntu-latest
         arch:
           - x64
+        include:
+          - version: 'nightly'
+            os: ubuntu-latest
+            arch: x64
+            continue-on-error: true
```

Comment:
Isn't that already set above in line 14?

Reply:
Yeah. I thought this would avoid the overall status to become red but that is apparently not possible